lib/ts/controllers/s-modal.ts

```diff
@@ -4,8 +4,7 @@ namespace Stacks {
         static targets = ["modal", "initialFocus"];
 
         private modalTarget!: HTMLElement;
-        private initialFocusTarget!: HTMLElement;
-        private hasInitialFocusTarget!: boolean;
+        private initialFocusTargets!: HTMLElement[];
```

Comment:
This change should be fine because Stimulus maps the return targets as an array even if there's only one, making this change api compatible with what we currently have.

No changes necessary.